lncrawl/core/cleaner.py

```diff
@@ -210,14 +203,14 @@ def clean_text(self, text) -> str:
         )
         return text
 
-    def extract_on_duplicate_sibling(self, tag: Tag):
+    def extract_on_duplicate_sibling(self, tag: TAG_LIKE):
```

Comment:
This is not actually necessary. These functions will always get called with `tag: Tag` type. Check the `clean_contents` method conditions:

```
  elif not isinstance(tag, Tag):
                continue  # Skip elements that are not a Tag
```